TurkishFilter/sections/specific.txt

```diff
@@ -43,6 +43,7 @@ fullhdfilmcibaba.com,720pizlemek.com,1080pfilmizle.net,fullhdfilmcidayi.com,hdfi
 !
 dizi6.plus###myModal
 dizi6.plus##body > a
+kimkazandi.com##div[class*="ads_"]
```

Comment:
It's better to add the domain to this rule
`gencgazete.net##div[class^="ads_"]`

Reply:
Okay.

Reply:
Fixed.